charts/gardener/controlplane/charts/runtime/templates/apiserver/deployment.yaml

```
@@ -274,9 +285,23 @@ spec:
         - name: gardener-apiserver-encryption-config
           mountPath: /etc/gardener-apiserver/encryption
         {{- end }}
+        {{- if not .Values.global.apiserver.tlsSecretName }}
+        {{- if not .Values.global.apiserver.etcd.tlsSecretName }}
```

Shouldn't this be:
```suggestion
        {{- if or (not .Values.global.apiserver.tlsSecretName) (not .Values.global.apiserver.etcd.tlsSecretName) }}
```
or something similar? This mount should not be needed only if *both* values above are non-empty, correct?